examples/text-generation/run_lm_eval.py

```diff
@@ -14,32 +14,33 @@
 # limitations under the License.
 
 ###############################################################################
-# Copyright (C) 2020-2021 Habana Labs, Ltd. an Intel Company
+# Copyright (C) 2020-2024 Habana Labs, Ltd. an Intel Company
 ###############################################################################
 
 import argparse
 import json
-import logging
 import multiprocessing as mp
 import os
 import time
 
-import lm_eval.evaluator
-import lm_eval.tasks
+from lm_eval import evaluator, tasks, utils
```

Comment:
```suggestion
from lm_eval import evaluator, utils
```

I do not see tasks is used anywhere!

Reply:
You are right, ``make style`` already fixed it